utils/lib/github-api-helpers.ts

```diff
@@ -51,23 +52,31 @@ export const fetchPaginatedGHResults = async (
   url: string,
   token: string
 ): Promise<GithubAPIPullRequestFile[]> => {
+  logger.debug(`Running fetch against ${url}`, { url });
   let files: GithubAPIPullRequestFile[] = [];
   let nextPageLink: string | null = url;
   try {
     while (nextPageLink) {
+      logger.debug(`Fetching ${nextPageLink}`, { url: nextPageLink });
       const resp = await fetch(nextPageLink, {
         headers: { authorization: `token ${token}` },
       });
+      // TODO: this should happen after the resp.ok check
       const responseJson = await resp.json();
 
       if (!resp.ok) {
+        logger.error(`Error from Github API`, {
+          url: nextPageLink,
+          status: resp.statusText,
+          code: resp.status,
+        });
         throw new Error(`Github API returned: ${responseJson.message}`);
```

Comment:
Would we want to also pass along the json response to the logger?

Reply:
jk, i realize that since it throws the error it'll display that in the workflows